src/utils.ts

```diff
@@ -25,7 +25,7 @@ export const nodeVersionGreaterThanEqual = (requestedVersion: string, version =
 };
 
 export function appendHeadersFromStore(
-  properties: { headers?: Record<string, string | Buffer | undefined> },
+  properties: { headers?: { [key: string]: Buffer | string | (Buffer | string)[] | undefined } },
```

Comment:
❓ Was the previous complaining?

Reply:
Yes but apparently for a different reason
Addressed here https://github.com/Workable/orka/pull/384/commits/c0576641c497705f62eabcac01d207b66903e27d